.ci_support/win_64_python3.9.____cpythonpython_implcpython.yaml

```diff
@@ -2,6 +2,8 @@ channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
+libprotobuf:
```

Comment:
This doesn't look right.

Reply:
Hmm, I may need to rerender.

Reply:
No, that doesn't look like it.

Reply:
@xhochy, any idea why conda-smithy doesn't notice that I've put version constraints on it?  Any suggestions for what I should change?

Reply:
The answer, near as I can discern from playing around is that conda-smithy behaves differently when `version` is included in the constraint than when it is not.